Changelog.md

```diff
@@ -4,6 +4,8 @@ Enhancements:
 
 * Add seed information to JSON formatter output. (#2388, Mitsutaka Mimura)
 * Include example id in the JSON formatter output. (#2369, Xavier Shay)
+* Allow changing `output_stream` after formatters have been setup and have
+  formatter output follow. (#2401, #2419, Ilya Lavrov)
```

Comment:
"after formatters have been setup and have formatter output follow" is worded a bit awkwardly, IMO. WDYT about this instead?

> Respect changes to `config.output_stream` after formatters have been instantiated. (#2401, #2419, Ilya Lavrov)